### PR TITLE
Run features in parallel sdk

### PR DESCRIPTION
**Pull request by karanshah-browserstack**

### CODE REVIEW

File: browserstack.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# =============================
+# Set BrowserStack Credentials
+# =============================
+# Add your BrowserStack userName and acccessKey here or set BROWSERSTACK_USERNAME and
+# BROWSERSTACK_ACCESS_KEY as env variables
+userName: BROWSERSTACK_USERNAME
+accessKey: BROWSERSTACK_ACCESS_KEY
+
```
Comment (francisf) [inline browserstack.yml:8]: source as a capability for attribution is missing?

Comment (karanshah-browserstack) [reply browserstack.yml:8]: Hardcoded inside capabilties rather than in yml file

File: pom.xml [hunk 2]
```diff
@@ -114,25 +114,35 @@
             <artifactId>browserstack-local-java</artifactId>
             <version>${browserstack.local.version}</version>
         </dependency>
+        <dependency>
```
Comment (kamal-kaur04) [inline pom.xml:117]: formatting..

File: src/test/java/com/browserstack/BrowserStackSerenityDriver.java [hunk 3]
```diff
@@ -1,61 +1,40 @@
...
-        }
-
+        File file = new File(getUserDir() + "/browserstack.yml");
+        browserStackYamlMap = convertYamlFileToMap(file, new HashMap<>());
+        userName = System.getenv("BROWSERSTACK_USERNAME") != null ? System.getenv("BROWSERSTACK_USERNAME") : (String) browserStackYamlMap.get("userName");
+        accessKey = System.getenv("BROWSERSTACK_ACCESS_KEY") != null ? System.getenv("BROWSERSTACK_ACCESS_KEY") : (String) browserStackYamlMap.get("accessKey");
+
+        MutableCapabilities capabilities = new MutableCapabilities();
+        HashMap<String, Object> bStackMap = new HashMap<>();
+        bStackMap.put("source", "serenity:sample-sdk:v1.0");
```
Comment (kamal-kaur04) [inline src/test/java/com/browserstack/BrowserStackSerenityDriver.java:33]: sdk?

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**